applications/integrationtests/geometryConcrete/CMakeLists.txt

```diff
@@ -31,11 +31,14 @@ if(ENABLE_PYTHON)
 endif()
 
 if(GMSH_PATH)
-    add_executable(GeometryConcreteAPI GeometryConcreteAPI.cpp)
+    add_executable(GeometryConcreteAPI EXCLUDE_FROM_ALL GeometryConcreteAPI.cpp)
```

Comment:
We should document some of the changes and the new features. This `EXCLUDE_FROM_ALL` removes the executable from `make all`, right? 
With those custom targets, all our executables have to be excluded from `all`. Is there a way to define that only once and not at every `add_executable` command?

Reply:
Yes, `EXCLUDE_FROM_ALL` removes it from the default build target. Afaik there is no way to set this globally. This is also part of the reason why I've introduced the `add_integrationtest` macro.